src/python/Utils/MathUtils.py

```diff
@@ -15,11 +17,11 @@ def quantize(inputVal, quanta):
     """
     if isinstance(inputVal, basestring):
         inputVal = float(inputVal)
-    elif not isinstance(inputVal, (int, float, long)):
```

Comment:
removing `long` here scares me. Perhaps we should override the long data type like:
```
if sys.version_info.major == 3:
  long = int
```
? Just an idea, I don't really know what the best way to approach it.

Reply:
If the plan is to drop python2, having a line that screams `past` was not so appealing :)

The "price" that we have to pay is to get accustomed to the new `int` type that is going to be "long" also in py2, which does not seem terrible. So I took the liberty to choose option 1. I am glad that you agree :)